lib/iiif_manifest/v3/manifest_builder/iiif_service.rb

```diff
@@ -13,11 +13,17 @@ def label=(label)
           inner_hash['label'] = label
         end
 
+        def summary
+          inner_hash['summary']
+        end
+
         def summary=(summary)
           return unless summary.present?
           inner_hash['summary'] = summary
         end
 
+        # TODO: issue 116: some other getters are missing, not needed?
```

Comment:
I think this will be more visible if it were converted into a github issue instead of inlining it here.